{{cookiecutter.repo_name}}/circle.yml

```diff
@@ -1,7 +1,3 @@
-general:
```

Comment:
There's no substantive change here; I've just updated the ordering to match recent services.